etcdutl/etcdutl/migrate_command.go

```diff
@@ -95,7 +95,11 @@ func (o *migrateOptions) Config() (*migrateConfig, error) {
 	c.be = backend.NewDefaultBackend(GetLogger(), dbPath)
 
 	walPath := datadir.ToWALDir(o.dataDir)
-	w, err := wal.OpenForRead(c.lg, walPath, walpb.Snapshot{})
+	lastSnapshot, err := getLastSnapshotIndex(walPath)
+	if err != nil {
+		return nil, fmt.Errorf(`failed to find last snapshot: %v`, err)
+	}
```

Comment:
Considering the past behaviour was to read from the start, should we fallback to `Index=0` in case we cannot find snapshot at a certain index?